src/pages/_ContentLoaded.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNamesBind from 'classnames/bind';
+import { ThemeContext } from '@cerner/terra-application/lib/theme';
+
+import styles from './ContentLoaded.module.scss';
+
+const cx = classNamesBind.bind(styles);
+
+const propTypes = {
+  /**
+   * The type of the content loaded
+   */
+  type: PropTypes.string,
+  /**
+   * The content to render
+   */
+  children: PropTypes.element.isRequired,
+};
+
+const ContentLoaded = ({ children, type }) => {
+  const theme = React.useContext(ThemeContext);
+
+  // Re enable if hash links aren't working
+  // useEffect(() => {
+  //   if (!window.location || window.location.length < 2) {
+  //     return;
+  //   }
+  //   const elementName = window.location.hash.slice(1);
+  //   const element = document.getElementById(elementName);
+  //   if (element) {
+  //     element.scrollIntoView();
+  //   }
+  // }, []);
```

Comment:
hash links appear to be working, but we previously had to use this to ensure the hash link was navigated to. I've left it for now until i'm satisfied we don't need it.

Reply:
> left it for now until i'm satisfied we don't need it.

left it = left it commented?  I just want to make sure we don't forget to delete it at some point.

Reply:
yeah, left it commented until i'm sure we don't need it.